### ISSUE TITLE
Uncatched ModelBuildingException

### ISSUE THREAD

**Issue by hohwille** [m-m-m/code#28]
```
java.lang.IllegalStateException: org.apache.maven.model.building.ModelBuildingException: 1 problem was encountered while building the effective model for xml-resolver:xml-resolver:1.2
[ERROR] 'distributionManagement.status' must not be specified. @ 
```

**Comment by hohwille** [comment]
Last log message was:
```
Reading effective model of /Users/hohwille/.m2/repository/xml-resolver/xml-resolver/1.2/xml-resolver-1.2.pom
```

**Comment by hohwille** [comment]
And indeed this POM is containing `distributionManagement.status`:
https://repo1.maven.org/maven2/xml-resolver/xml-resolver/1.2/xml-resolver-1.2.pom

So maven central contains a POM that maven itself is unable to parse.

**Comment by hohwille** [comment]
Fixed. Will be released with `1.0.0-beta5`.
